wordpress/client.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import wordpress
+
+class APIClient:
+    api_path = "wp-json/wp/v2"
+    user = wordpress.wp_user
+    password = wordpress.wp_password
+    api_url = f'{wordpress.wp_host}/{api_path}'
```

Comment:
again, I think restricting this class to only have a single instance it's an unnecessary constraint it may cause us trouble in the future.

Reply:
Definetely should be instance attributes. thanks

Reply:
Should be also changed probably in Koiki wcfmmp.py